### PR TITLE
WIP Feature/42 

### PR DESCRIPTION
**Pull request by ynynl**
#42 

TODO: test

Completed:
image

### CODE REVIEW

File: web-client/src/shared/components/drivers-registration/PreScreenRequirements/index.tsx [hunk 1]
```diff
@@ -10,10 +10,8 @@ import { PreScreen } from 'shared/redux/driverRegistration';
 import { useSelector } from 'react-redux';
 import { RootState } from '../../../redux/index'
 
-import FormInput from './FormInput'
-
 import { policeRecordsCheckOptionsLabel, policeRecordsCheckTypeLabel, drivingAbstractOptionsLabel, LicenseAndVehicleLabel, LicenseClasses } from './label';
-
+import FormInput from '../FormInput';
```
Comment (l2silver) [inline web-client/src/shared/components/drivers-registration/PreScreenRequirements/index.tsx:14]: Is this being used?

Comment (l2silver) [inline web-client/src/shared/components/drivers-registration/PreScreenRequirements/index.tsx:14]: Woops! How did we miss this?

File: web-client/src/shared/components/drivers-registration/Agreement/index.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
+import React from 'react';
```
Comment (l2silver) [inline web-client/src/shared/components/drivers-registration/Agreement/index.tsx:1]: I love how you did this! Excellent work.

**Review by l2silver — COMMENTED** [review]
Wow, great work! My only comment really is, do you think we could add a couple of tests to give us some coverage!

**Comment by l2silver** [comment]
- [ ] Add tests

File: web-client/src/shared/components/drivers-registration/PreScreenRequirements/index.tsx (hunk 1 above)
Comment (ynynl) [reply web-client/src/shared/components/drivers-registration/PreScreenRequirements/index.tsx:14]: I put it in the same folder of the prescreen component, and to reuse it in this ticket I moved it out

**Review by l2silver — APPROVED** [review]
Thanks!

**Pull request merged**